SPRINTLOG.md

```diff
@@ -231,11 +231,11 @@ Please add a _short_ line describing the PR you make, if the PR implements a spe
 
 # 2023-03-17 - 2023-03-31
 
-_Nothing merged in CLI during this sprint_
```

Comment:
I don't know if we should retrospectively change the SPRINTLOG (as in genuinely don't know, but it feels slightly off to edit previous entries).

Reply:
I agree, but this is incorrect since this sprintlog is for the web/api, so this is just a typo / left over when copy pasting.

Reply:
So when reading this I got confused when it said "CLI"